crates/sui-core/src/authority.rs

```diff
@@ -701,18 +703,42 @@ impl AuthorityState {
         transaction: VerifiedTransaction,
         epoch_store: &Arc<AuthorityPerEpochStore>,
     ) -> SuiResult<VerifiedSignedTransaction> {
-        let (_gas_status, input_objects) = sui_transaction_checks::check_transaction_input(
+        let tx_digest = transaction.digest();
+        let tx_data = &transaction.data().intent_message().value;
+
+        let input_object_kinds = tx_data.input_objects()?;
+        let receiving_objects_refs = tx_data.receiving_objects();
+
+        sui_transaction_checks::deny::check_transaction_for_signing(
```

Comment:
I don't understand which package load you're referring to?

Reply:
The package loaded during deny check, we discussed this yesterday

Reply:
ah, right - i'll add a comment, but its probably not worth optimizing too much, since this code isn't reached when the package deny map is empty, and the package loads are cached anyway.